src/patternfly/components/Tile/tile.scss

```
@@ -70,22 +69,17 @@
     --pf-c-tile__title--Color: var(--pf-c-tile--m-disabled__title--Color);
     --pf-c-tile__body--Color: var(--pf-c-tile--m-disabled__body--Color);
     --pf-c-tile--before--BorderWidth: 0;
+    --pf-c-tile__icon--Color: var(--pf-c-tile--m-disabled__header--m-stacked__icon--Color);
```

could this be renamed?

```suggestion
    --pf-c-tile__icon--Color: var(--pf-c-tile--m-disabled__icon--Color);
```